### DISCUSSION TITLE
bg_color will only accept  rgb/hex

### DISCUSSION THREAD

**Discussion by strelniece** [Clooos/Bubble-Card#314] [Feature requests]
Trying to use my css-variables in bg_color in custom:bubble-pop-up card, but it seems only rgb or hex values works. Not even names (ie red) is accepted.

the "old" way setting the color via styles will accept variables.

using v1.5.0, tried both gui and yaml-mode

**Comment by Clooos** [comment]
Indeed! I really need to add this possibility. Thanks for your feedback!

**Comment by mkrus** [comment]
any way of getting the default "system/theme" colours? or at least be able to pick different colours based on the dark/light mode?

**Comment by Clooos** [comment]
It's the default theme color by default, in the GUI editor if your remove it it will go back to its default.
